bcda/cclf/cclf_test.go

```diff
@@ -24,22 +25,25 @@ import (
 	"github.com/CMSgov/bcda-app/bcda/models"
 )
 
-const BASE_FILE_PATH = "../../shared_files/"
-
-var origDate string
-
 type CCLFTestSuite struct {
 	suite.Suite
 	pendingDeletionDir string
+
```

Comment:
Should we remove the extraneous whitespace?

Reply:
Was actually using it to separate like configurations. so basePath and the associated path are grouped together